src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
   const [releaseDate, setReleasedate] = useState(
     editableMode ? formData?.release_date : ``
   );
+  return (
+    <Formik
+      initialValues={{
+        genres,
+        title,
+        overview,
+        posterPath,
+        runtime,
+        releaseDate,
+      }}
+      validationSchema={validationSchema}
+      onSubmit={(values) => {
+        onModalSubmit({
+          ...formData,
+          genres: values.genres,
+          title,
+          overview,
+          poster_path: posterPath,
+          runtime,
+          release_date: releaseDate,
+        });
+      }}
+    >
+      {({ submitForm }) => (
+        <Form>
+          <>
+            <FormikTextField
+              label="Title"
+              name="title"
+              variant="filled"
+              color="secondary"
+              className={classes.formControl}
+              InputProps={{ className: classes.title }}
+              InputLabelProps={{
+                classes: {
+                  root: classes.label,
+                  focused: classes.focusedLabel,
+                },
+              }}
+              value={title}
+              onChange={(e) => setTitle(e.target.value)}
+            />
 
-  const handleChangeGenres = (event) => {
-    setGenres(event.target.value as string[]);
-  };
-
-  const handleChangeTitle = (event) => {
-    setTitle(event.target.value);
-  };
-
-  const handleChangeOverview = (event) => {
-    setOverview(event.target.value);
-  };
-
-  const handleChangeMovieUrl = (event) => {
-    setMovieUrl(event.target.value);
-  };
-
-  const handleChangeRuntime = (event) => {
-    setRuntime(+event.target.value);
-  };
+            <FormikTextField
+              id="date"
+              label="RELEASE DATE"
+              type="date"
+              variant="filled"
+              name="releaseDate"
+              color="secondary"
+              className={classes.formControl}
+              InputProps={{ className: classes.title }}
+              InputLabelProps={{
+                classes: {
+                  root: classes.label,
+                  focused: classes.focusedLabel,
+                },
+                shrink: true,
+              }}
+              value={releaseDate}
+              onChange={(e) => setReleasedate(e.target.value)}
+            />
 
-  const handleChangeReleaseDate = (event) => {
-    setReleasedate(event.target.value);
-  };
+            <FormikTextField
+              label="Movie URL"
+              name="posterPath"
```

Comment:
change `posterPath` and `releaseDate` to `poster_path` and `release_date` since you won't need `useState` variables.
Also do the same for validation schema.